queries.sql

```diff
@@ -64,3 +64,21 @@ SELECT MAX(escape_attempts) FROM animals;
 SELECT MAX(weight_kg), MIN(weight_kg) FROM animals GROUP BY species;
 SELECT AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species;
 
+/* write queries to answer the questions */
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.id = 4;
```

Comment:
We shouldn't use the owner's id to retrieve this kind of data because when the DB holds many records as it's designed for, you would need to find the id within those records and fetch the data. That would be pretty inefficient and time-consuming. Please try to get the data by the name of the owner 🤓 You could make a left join between owners and animals.owner_id and then try something like 
```sql
WHERE full_name = 'Melody Pond';
```

Please make the necessary changes so that your project meets the requirements ✔

Reply:
Done.